src/packs/ranged/Thunder Gun.yml

```diff
@@ -1,16 +1,13 @@
 _id: bfAf7GaaEuZf0k9t
 data:
-  ab: 0
+  ab: 1
   ammo:
     burst: false
     longReload: false
     max: 6
     suppress: false
     type: typeAPower
     value: 6
-  bundle:
-    amount: 1
-    bundled: false
   cost: 1000
```

Comment:
Don't update or remove data unrelated to PR

Reply:
This one was handled in #34.